src/components/forms/HPButton.tsx

```diff
@@ -1,8 +1,27 @@
 import * as React from 'react';
-import { Fab } from '@material-ui/core';
+import { Button } from '@material-ui/core';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
 
-export const HPButton = ({ onClick, label }) => {
-    return <Fab onClick={onClick} variant="extended" style={{marginLeft: "10px", marginTop: "10px", backgroundColor: "red", color: "white"}}>
-        {label}
-    </Fab>
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    button: {
+      margin: theme.spacing(1),
+      backgroundColor: 'red',
+      color: 'white',
+      '&:hover': {
+        backgroundColor: '#ff5252'
+      }
+    },
+    input: {
```

Comment:
what's this `input` section for? it's not explicitly referenced anywhere

Reply:
Double checking now. That may have been a lack of due diligence on my part when copying over a solution from Stackoverflow.